dev/ci/docker_linux/Gemfile

```diff
@@ -2,7 +2,3 @@
 # in dev/ci/docker_linux/Dockerfile to retrigger a Docker build on Cirrus.
 
 source "https://rubygems.org"
-
```

Comment:
This PR looks good to me.

As a nit, can you also update the timestamp on the Dockerfile https://github.com/flutter/flutter/blob/master/dev/ci/docker_linux/Dockerfile#L13, to bust the cache and trigger a rebuild of the Dockerfile. I want to make sure that all the linux tests pass without fastlane installed.

Reply:
Updated.